package.json

```diff
@@ -77,7 +77,7 @@
     "wdio-default": "terra wdio",
     "wdio-lowlight": "terra wdio --themes clinical-lowlight-theme",
     "wdio-fusion": "terra wdio --themes orion-fusion-theme",
-    "wdio": "terra wdio --themes terra-default-theme clinical-lowlight-theme orion-fusion-theme",
+    "wdio": "terra wdio --themes terra-default-theme clinical-lowlight-theme orion-fusion-theme --spec packages/terra-button/tests/wdio/button-spec.js --browsers chrome --gridUrl grid314.test.devcernerpowerchart.com",
```

Comment:
I think you want to discard this change

Reply:
Right, I've added it by mistake. Thank you!